OjasWebApp/src/com/MyProject.java

```diff
@@ -0,0 +1,7 @@
+package com;
+
+public class MyProject {
+   public void disp() {
```

Comment:
Please change the method name to Display

Reply:
Done